README.md

```diff
@@ -2,8 +2,8 @@
 Open Source Programs (OSP) is an application that simplifies the processing and selection procedure of Open Source Programs of AnitaB.org Open Source or other third-party programs. This is the Backend repo for OSP.
 
 ## Tech Stack
-- Django
-- Django REST Framework
+- Django 3.0.7
```

Comment:
IMO we should mention the python version as well

Reply:
Yes same. @vj-codes Please make the change.

Reply:
python 3.8.3 in after rest frame right? @bismitaguha @Monal5031

Reply:
3.7 or 3.7+ should work. @Monal5031 your opinion.

Reply:
Agreed. 3.7+ should be good